pkg/network/payload/payload.go

```diff
@@ -7,3 +7,22 @@ type Payload interface {
 	EncodeBinary(io.Writer) error
 	DecodeBinary(io.Reader) error
 }
+
+// NullPayload is a dummy payload with no fields
```

Comment:
nit: dot in the end

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
I will start putting dots in the end of sentences.